### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -20,8 +20,8 @@ unstable = [
 
 [dependencies]
 matches = "0.1.2"
-html5ever = "0.4"
-hyper = {version = "0.7", optional = true}
+html5ever = ">= 0.4, <=0.5"
```
Comment (SimonSapin) [inline Cargo.toml]: Do you know if `<=0.5` the same as `<=0.5.0` or `<0.6.0` ? I think we want the latter, since the former excludes e.g. 0.5.1.

Comment (untitaker) [inline Cargo.toml]: You're right, updated.

**Comment by SimonSapin** [comment]
Thanks!

**Comment by untitaker** [comment]
Could you cut a new minor version for this?

**Comment by SimonSapin** [comment]
Just published v0.3.2.

**Pull request merged**